src/planscape/config/treatment_goals.json

```diff
@@ -563,52 +563,29 @@
           "questions": [
             {
               "id": 13,
-              "long_question_text": "Where are the best opportunities to effectively reduce high severity fire risk to riparian habitats",
+              "long_question_text": "Where are the best opportunities to reduce high severity fire risk to riparian habitats?",
               "short_question_text": "Reduce Fire Risk to Riparian Habitat",
               "scenario_output_fields_paths": {
                 "pillars": [
-                  "biodiversity_conservation",
                   "fire_dynamics",
-                  "economic_diversity",
-                  "social_and_cultural_wellbeing",
-                  "wetland_integrity",
-                  "fire_adapted_communities"
+                  "forsys"
                 ],
                 "elements": [
-                  "species_diversity",
-                  "functional_fire",
                   "severity",
-                  "focal_species",
-                  "wood_product_industry",
-                  "environmental_justice",
-                  "composition",
-                  "hazard"
+                  "Forsys"
                 ],
                 "metrics": [
-                  "threatened_endangered_vertebrate_species_richness",
-                  "california_spotted_owl",
-                  "predicted_ignition_probability_human_caused",
-                  "annual_burn_probability",
-                  "mountain_lion",
-                  "coastal_california_gnatcatcher",
-                  "mountain_yellow_legged_frog",
-                  "california_red_legged_frog",
-                  "joshua_tree",
-                  "hermes_copper_butterfly",
-                  "cost_of_potential_treatments",
-                  "tribal_land_designations",
-                  "aquatic_species_richness",
-                  "structure_exposure_score",
-                  "housing_unit_density",
-                  "probability_of_fire_severity_high"
+                  "probability_of_fire_severity_high",
+                  "cwhr_vegetation_type"
                 ]
               },
               "scenario_priorities": [
                 "probability_of_fire_severity_high"
               ],
               "weights": [],
               "stand_thresholds": [
-                "probability_of_fire_severity_high > 0"
+                "probability_of_fire_severity_high > 0",
+                "!(cwhr_vegetation_type  %in% c(59,79, 43,5,21,56,15,41,35,37,19))"
```

Comment:
this ended up being kind of the same as not using the reclassified layer... not sure if I missed something.
Used https://www.statology.org/not-in-r/  for this instead of crazy ORs